source/concepts/environments/instance-scheduling.html.md.erb

```diff
@@ -18,9 +18,9 @@ review_in: 6 months
 
 ## Feature description
 
-This feature automatically stops non-production EC2 and RDS instances overnight, in order to save on AWS costs and reduce environmental impact. Stopped instances don't incur charges, but Elastic IP addresses or EBS volumes attached to those instances do.
+This feature automatically stops non-production EC2 and RDS instances overnight and over each weekend, in order to save on AWS costs and reduce environmental impact. Stopped instances don't incur charges, but Elastic IP addresses or EBS volumes attached to those instances do.
 
-The instances will be automatically stopped every weekday at 9pm night and started at 6am in the morning. By default, this includes every EC2 and RDS instance in every non-production environment (development, test, pre-production) without requiring any configuration from the end user. Users can customise the default behaviour by attaching the `instance-scheduling` tag to EC2 and RDS instances with one of the following values:
+The instances will be automatically [stopped each weekday at 21:00](https://github.com/ministryofjustice/modernisation-platform/blob/19a7e48b366cfbb9d24c30f4620b12df886baa8e/terraform/environments/core-shared-services/instance-scheduler-lambda-function.tf#L35) and [started at 06:00 each weekday](https://github.com/ministryofjustice/modernisation-platform/blob/19a7e48b366cfbb9d24c30f4620b12df886baa8e/terraform/environments/core-shared-services/instance-scheduler-lambda-function.tf#L61) morning, which includes shut down on Friday night and startup on Monday morning. By default, this includes every EC2 and RDS instance in every non-production environment (development, test, preproduction) without requiring any configuration from the end user. Users can customise the default behaviour by attaching the `instance-scheduling` tag to EC2 and RDS instances with one of the following values:
```

Comment:
I think these would be better as `main` rater than a specific branch. Otherwise on review / use these can potentially point at outdated code.

Reply:
If the preference is to linking only to `main` branch, I'll remove any links that point to specific lines, as they will drift as code changes and become confusing.